src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,7 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Hi Hi, you are missing header comment here. Remember, all non trivial classes/methods should have javadoc format header comments.